proto/streamlit/proto/DocString.proto

```diff
@@ -18,19 +18,35 @@ syntax = "proto3";
 
 // Formatted text
 message DocString {
-
-  // The name of the object.
+  // The name the user gave to the variable holding this object.
   string name = 1;
```

Comment:
Whoops, my bad for not noticing this earlier, especially given that I have code ownership over these files specifically to enforce this 😅 

We'll need to ensure that these proto changes are backwards compatible given that the SiS team relies on them

We also want to be a bit more strict than normal proto backwards compatibility guidelines to also avoid changing field names to make things easier on folks when they're upgrading the OS lib version.

Reply:
Done. PTAL.